pyproject.toml

```diff
@@ -17,7 +17,8 @@ classifiers = [
     "License :: OSI Approved :: Apache Software License",
 ]
 dependencies = [
-    "triton>=2.2.0",
+    "triton>=2.2.0, <3.0.0",
+    "torch>=2.1.2",
     "transformers>=4.40.2"
```

Comment:
FYI: version spec and version entry in pyproject.toml
https://peps.python.org/pep-0508/
https://peps.python.org/pep-0631/